assets/helpers/payPalExpressCheckout/payPalExpressCheckout.js

```diff
@@ -2,32 +2,23 @@
 
 // ----- Imports ----- //
 
-import { payPalExpressError } from 'helpers/payPalExpressCheckout/payPalExpressCheckoutActions';
+import { logException } from 'helpers/logger';
 import { routes } from 'helpers/routes';
-import type { PageState, CombinedState } from 'helpers/payPalExpressCheckout/payPalExpressCheckoutReducer';
-
+import type { Csrf as CsrfState } from 'helpers/csrf/csrfReducer';
+import type { Currency } from '../internationalisation/currency';
 
 // ----- Functions ----- //
 
-const loadPayPalExpress = () => new Promise((resolve) => {
-
-  if (!window.paypal) {
```

Comment:
This check is no longer required, because we are using hasLoaded to check whether or not to call this function